docs/source/developer_guide/guides/1_simple_python_stage.md

```diff
@@ -230,6 +230,19 @@ Before constructing the pipeline, we need to do a bit of environment configurati
 ```python
 configure_logging(log_level=logging.DEBUG)
 ```
+We use the default configuration with the `DEBUG` logging level. The logger will output to both the console and a file. The logging handlers are non-blocking since they utilize a queue to send the log messages on a separate thread.
+
+We can also use `configure_logging` to add one or more logging handlers to the default configuration. The added handlers will also be non-blocking. The following is from the
+[Grafana example](../../../../examples/digital_fingerprinting/production/grafana/README.md) where we add a [Loki](https://grafana.com/oss/loki/) logging handler to also publish Morpheus logs to a Loki log aggregation server.
+```python
+loki_handler = logging_loki.LokiHandler(
+    url=f"{loki_url}/loki/api/v1/push",
+    tags={"app": "morpheus"},
+    version="1",
+)
+
+configure_logging(loki_handler, log_level=log_level)
```

Comment:
Containing the documentation to this one example is going to make it hard to discover. Can we move this to a more general location? Possibly under `extra_info`

Reply:
Created issue #1520